runtime/compiler/runtime/DataCache.hpp

```diff
@@ -41,13 +42,16 @@ class TR_DataCache
    uint8_t         *_allocationMark; // used if we want to give back memory up to previously set mark
    //TR::Monitor       *_mutex;     // Is this needed?
    int32_t          _status;    // mostly RAS at this point
+   OMR::RSSRegion  *_rssRegion; // to report footprint
 public:
    enum {
       RESERVED=1,
       ACTIVE,
       ALMOST_FULL,
       FULL,
    };
+
+   TR_DataCache() : _rssRegion(NULL) {}
```

Comment:
The constructor will not be called. It's better to initialize this field at the point where all the others are initialized.

Reply:
Done